src/v/cluster/metadata_cache.cc

```diff
@@ -318,6 +319,12 @@ metadata_cache::get_default_record_value_subject_name_strategy() const {
     return pandaproxy::schema_registry::subject_name_strategy::topic_name;
 }
 
+std::optional<std::chrono::milliseconds>
+metadata_cache::get_default_delete_retention_ms() const {
+    // TODO: return config::shard_local_cfg().tombstone_retention_ms();
```

Comment:
why not add the configuration option before this commit so we don't need to have a TODO item?